src/reflect/scala/reflect/internal/Phase.scala

```diff
@@ -41,9 +41,8 @@ abstract class Phase(val prev: Phase) {
   def checkable: Boolean = true
 
   // NOTE: sbt injects its own phases which extend this class, and not GlobalPhase, so we must implement this logic here
-  private var _erasedTypes = ((prev ne null) && (prev ne NoPhase)) && (prev.name == "erasure" || prev.erasedTypes)
-  protected def erasedTypes_=(value: Boolean): Unit = {_erasedTypes = value}
-  final def erasedTypes: Boolean = _erasedTypes // overridden in back-end
+  private val _erasedTypes = ((prev ne null) && (prev ne NoPhase)) && (prev.name == "erasure" || prev.erasedTypes)
+  final def erasedTypes: Boolean = _erasedTypes
```

Comment:
At this point I don't any difference to just having `private val erasedTypes = ...` - or is there?

Reply:
You're right, updated.